<launch>
    <node name="image_server" pkg="control_demo" type="image_server" output="screen" />
</launch>
